### PR TITLE
remove PerformanceEntryFilterOptions

### PR DESCRIPTION
**Pull request by igrigorik**
Background: https://github.com/w3c/performance-timeline/issues/57#issuecomment-248292074. Closes https://github.com/w3c/performance-timeline/issues/60.

Preview: https://cdn.rawgit.com/w3c/performance-timeline/remove-filteroptions/index.html

### CODE REVIEW

**Comment by igrigorik** [comment]
@toddreifsteck @plehegar PTAL.

File: index.html [hunk 1]
```diff
@@ -303,85 +300,31 @@
...
       <p>
         The <dfn for="Performance">getEntriesByName</dfn> method
         returns a <a>PerformanceEntryList</a> object returned by
-        <a for='Performance' data-lt='getEntries'>getEntries({'name': name})</a>
-        if optional `entryType` is omitted, and
-        <a for='Performance' data-lt='getEntries'>getEntries({'name': name, 'entryType': type})</a>
-        otherwise.
+        <a href="#filter-performance-entry-buffer-by-name-and-type"></a>
+        algorithm with <var>name</var> set to `name` and <var>type</var> set to
+        `null` if optional `entryType` is omitted, and <var>type</var> set to
```
Comment (plehegar) [inline index.html]: there is no type argument for getEntriesByName. Should say:
 ... with <var>name</var> set to `name` and <var>type</var> set to  `null`.

Comment (plehegar) [inline index.html]: scratch that. not enough coffee it seems.

**Comment by igrigorik** [comment]
Thanks Philippe! 👍

**Comment by toddreifsteck** [comment]
Nicely done! Thanks for doing this, @plehegar and @igrigorik

**Pull request merged**